learn/avalanche-bridge-faq.md

```diff
@@ -152,6 +162,10 @@ While using tx.origin to check authorization within smart contracts poses potent
 
 No single party has access to the SGX enclave address. Only the enclave itself can construct/sign a transaction using that key when it receives approvals from 3 of 4 wardens. In this sense, the enclave here is functioning as a cross-chain smart contract.
 
+#### Why doesn't the bridge hold funds in a a smart contract?
+
+Not using a smart contract allows for lower gas fees and faster transfers. 
```

Comment:
Maybe something like "Avoiding the use of a smart contracts, dramatically simplifies the end-to-end transfer requirements resulting in lower gas fees and faster transfer resolutions"